### ISSUE TITLE
How to skip the "Welcome to Samsung Internet" prompt

### ISSUE THREAD

**Issue by PicoCreator** [SamsungInternet/chromedriver-examples#1]
I have tried using the examples with 'com.sec.android.app.sbrowser', on an updated samsung internet 11.2.1.3 on a Galaxy S8. And the respective chromedriver

For most part it works except that at the start of the session - the prompt of "Welcome to Samsung Internet" and "Set default browser" appears and require manual clicks before being able to proceed.

How do I configure these prompts to be skipped? As this makes it near impossible for a remote automation use case.

**Comment by soulgalore** [comment]
I'm also stuck at the prompt (the navigation never happens):
<img width="444" alt="Screenshot 2020-12-02 at 11 58 19" src="https://user-images.githubusercontent.com/540757/100864028-b49ea780-3495-11eb-8690-8a3f1516bc0b.png">

I'm running the duckduckgo example (updated to use Chromedriver 83, using Samsung Internet 13.0.1.64).

**Comment by usagikengood** [comment]
For me, I resolve it by first switching to native mode, detect if this page is popped up, and dismiss it , and then switch it back to browser mode.

**Comment by soulgalore** [comment]
Native/browser mode is a Appium thing right? I think it would be cool if it would work with just Selenium. Thinking it should have switches as many Chromium things do, so you can just add those switches on startup to remove the welcome screen (and setting Samsung as default browser).

**Comment by usagikengood** [comment]
> Native/browser mode is a Appium thing right? I think it would be cool if it would work with just Selenium. Thinking it should have switches as many Chromium things do, so you can just add those switches on startup to remove the welcome screen (and setting Samsung as default browser).

yes it's Appium side thing, as those popups are native app thing. I agree with you if there are switches to stop it.

**Comment by Oktaliem** [comment]
@PicoCreator I use Appium and set capability "capabilities.setCapability("noReset", "true");". It works , but  you will find "NoSuchElementException" everywhere when you run the test with Appium. I wish somebody that has successfully ran Appium test with "Samsung Internet" can give us advise here.

**Comment by soulgalore** [comment]
Hey SamsungInternet,  is there a way for you to fix the prompt issue? 

Also is there a reason that you use  another device socket for devtools, it would be sweat if we could use Chromedriver just as is.

**Comment by cateyelow** [comment]
@soulgalore It's simple for solving that issue Combine uiautomator2 then click :3
